src/templates/status.html

```diff
@@ -27,58 +49,205 @@
         })
     }
 
-    function getRunStatus(runId, cid, queryToken) {
-        $.ajax(
-            {
-                type: "get",
-                url: `{{ broadway_api_url }}/grading_run_status/${cid}/${runId}`,
-                headers: buildHeader(queryToken),
-                success: function (result) {
-                    let status = Object.values(result["data"]["student_jobs_state"])[0]
-                    let jobId = Object.keys(result["data"]["student_jobs_state"])[0]
-                    var status_map = {
-                        "scheduled": "btn-info",
-                        "running": "btn-info",
-                        "success": "btn-success",
-                        "failed": "btn-danger",
-                        "": "btn-info"
-                    };
+    /**
+     * Asyncronously gets the job id and status of a run. Assumes this run only has 1 job in it (which
+     * is true for student triggered pre-deadline runs). Once got the info, passes them into callback 
+     * functions within a signle object parameter. Example successCallback:
+     *     function onSuccess({ runId, cid, status, jobId, queryToken, <fields you passed in as "extraParams"> }) {
+     *         console.log(`Grading job with id ${jobId}, has status ${status}`)
+     *     }
+     * Which is equivalent to 
+     *     function onSuccess(params) {
+     *         console.log(`Grading job with id ${params.jobId}, has status ${params.status}`)
+     *     }
+     * See `getRunStatus` as an example usage.
+     * This function is created so that multiple other functions can query the same endpoint.
+     */
```

Comment:
```suggestion
    /**
     * Asynchronously gets the job id and status of a run. Assumes this run only has 1 job in it (which
     * is true for student triggered pre-deadline runs). After getting the info, it is passed into callback 
     * functions with a single object parameter. Example successCallback:
     *     function onSuccess({ runId, cid, status, jobId, queryToken, <fields you passed in as "extraParams"> }) {
     *         console.log(`Grading job with id ${jobId}, has status ${status}`)
     *     }
     * Which is equivalent to 
     *     function onSuccess(params) {
     *         console.log(`Grading job with id ${params.jobId}, has status ${params.status}`)
     *     }
     * See `getRunStatus` as an example usage.
     * This function is created so that multiple other functions can query the same endpoint.
     */
```
nitpick but small typos/grammar I noticed.